### DISCUSSION TITLE
LDAP login broken after 2024-03-26 release

### DISCUSSION THREAD

**Discussion by pilot7777** [minio/minio#19535] [Q&A]
I saw there were LDAP changes in 2024-03-30 relase. Now it returns "invalid Login" when using LDAP. It worked fine with 2024-03-26 release. Here is our LDAP settings. I don't see any LDAP error in journalctl output. Thanks for any help!

MINIO_IDENTITY_LDAP_SERVER_ADDR="ldaps.tktm.io"
MINIO_IDENTITY_LDAP_LOOKUP_BIND_DN="CN=our_username,OU=ServiceAccounts,OU=Techops,DC=techops,DC=info"
MINIO_IDENTITY_LDAP_LOOKUP_BIND_PASSWORD="our_password"
MINIO_IDENTITY_LDAP_USER_DN_SEARCH_BASE_DN="OU=Techops,DC=techops,DC=info"
MINIO_IDENTITY_LDAP_USER_DN_SEARCH_FILTER="(&(objectCategory=user)(sAMAccountName=%s))"
MINIO_IDENTITY_LDAP_GROUP_SEARCH_BASE_DN="OU=Techops,DC=techops,DC=info"
MINIO_IDENTITY_LDAP_GROUP_SEARCH_FILTER="(&(objectClass=group)(member=%d))"

**Comment by harshavardhana** [comment]
yes it won't work due to LDAP normalization.

**Reply by pilot7777** [reply]
How do I make it work? I checked the DOCs but it does not look like they have been updated.

**Reply by harshavardhana** [reply]
Wait for next release where `mc admin cluster iam export/import` will fix it automatically

**Comment by beeritis** [comment]
Just suffered the same problem.  Got a two node - single endpoint setup with replication enabled.

Updated one node , s3 etc working fine but broke LDAP auth.  This was using the 20240605 releaase (latest as of today) . 

Get this error in the logs : 

```
Error: STS credential could not be verified: signature is invalid (*fmt.wrapError)
```

Looks as though it tries to use STS rather than LDAP to authenticate. 

Don't know if there are LTS releases as this core functionality shouldn't be breaking!
